### PR TITLE
specify version of django-allauth that does not break on build.

### LINKED ISSUE #537: specify version of django-allauth.
## Problem
Currently the requirements file does not specify version of ```django-allauth``` to be installed so it installs the latest version, but that breaks when building the project in local environment.

## Solution
Specify version that works when building the project and does not break.

### LINKED ISSUE #573: Pathway didn't work
when the user chooses pathway other than pathway number 1, the pathway didn't show anything, only a loading icon.

### PR DESCRIPTION
**Pull request by sonlinux**
fix #537

### CODE REVIEW

File: deployment/docker/REQUIREMENTS.txt [hunk 1]
```diff
@@ -23,7 +23,7 @@ setuptools==18.5
 WeasyPrint
 requests<2.12
 beautifulsoup4
-django-allauth
+django-allauth==0.33.0
```
Comment (cchristelis) [inline deployment/docker/REQUIREMENTS.txt]: I would prefer us to specify this as: `django-allouth>=0.33,<0.34` minor point releases are made to fix critical bugs like security as well as just overall niceness...

**Review by cchristelis — CHANGES_REQUESTED** [review]
Hi @sonlinux, please do a pip freeze on your container and specify versions for the remaining Requirements, please. I don't want us to get to the same issue again.

File: deployment/docker/REQUIREMENTS.txt (hunk 1 above)
Comment (sonlinux) [reply deployment/docker/REQUIREMENTS.txt]: Thanks @cchristelis 
I updated.

**Comment by sonlinux** [comment]
I updated @cchristelis 
Fixed.
Thanks

File: deployment/docker/REQUIREMENTS.txt [hunk 2]
```diff
@@ -1,33 +1,82 @@
...
+lazy-object-proxy==1.3.1
+Markdown==2.6.11
+MarkupSafe==1.0
+mccabe==0.6.1
+more-itertools==4.1.0
+nodeenv==1.2.0
+oauthlib==2.0.6
+packaging==17.1
+pdfrw==0.4
+pep8==1.7.1
```
Comment (dimasciput) [inline deployment/docker/REQUIREMENTS.txt:55]: @sonlinux, You don't need this in production, please put these in REQUIREMENTS-DEV.txt

**Comment by dimasciput** [comment]
I'll fix it in another PR, closing this PR for now

**Pull request closed without merging**